chain/client/src/chunk_executor_actor.rs

```diff
@@ -766,3 +766,761 @@ fn get_receipt_proofs_for_shard(
         .map(|res| res.map(|kv| kv.1))
         .collect()
 }
+
+#[cfg(test)]
```

Comment:
this file is getting pretty big, so I suggest moving tests the dedicated file like `src/tests/chunk_executor_actor.rs`, we already use this pattern in multiple places such as `runtime/runtime/src/tests/apply.rs` for example